source/appModules/outlook.py

```diff
@@ -332,6 +332,24 @@ def _generateTimeRangeText(self,startTime,endTime):
 		# Translators: a message reporting the time range (i.e. start time to end time) of an Outlook calendar entry
 		return _("{startTime} to {endTime}").format(startTime=startText,endTime=endText)
 
+	@staticmethod
+	def _generateCategoriesText(appointment):
+		categories = appointment.Categories
+		if not categories:
+			return None
+		if len(categories.split("; ")) > 1:

```

Comment:
According to <https://docs.microsoft.com/en-us/office/vba/api/outlook.appointmentitem.categories> the separator is saved in the registry and may differ depending on the user's locale. Could you rewrite this to fetch the separator from the registry and cache it to avoid a lookup for every appointment.

Reply:
I decided to remove category count detection. It isn't worth this extra complexity. Also note that, when provided in a list view, the column header is spoken as "categories", even when there's only one.